Raw json:

{'hasError': True, 'errParameter': ['fs.s3a.connection.timeout', 'fs.s3a.threads.max', 'fs.s3a.select.input.csv.comment.marker', 'hadoop.http.authentication.token.validity', 'fs.client.resolve.remote.symlinks', 'dfs.client.ignore.namenode.default.kms.uri'], 'reason': ['fs.s3a.connection.timeout: The value of this property is not a valid integer.', 'fs.s3a.threads.max: The value of this property is not a valid integer.', 'fs.s3a.select.input.csv.comment.marker: The value of this property is not a valid string.', 'hadoop.http.authentication.token.validity: The value of this property is not a valid integer.', 'fs.client.resolve.remote.symlinks: The value of this property is not a valid boolean.', 'dfs.client.ignore.namenode.default.kms.uri: The value of this property is not a valid boolean.']}

{'hasError': True, 'errParameter': ['fs.s3a.connection.timeout'], 'reason': ['The value of fs.s3a.connection.timeout is not a valid integer.']}

{'hasError': True, 'errParameter': ['fs.s3a.connection.timeout'], 'reason': ['fs.s3a.connection.timeout is not a valid property for hcommon version 3.3.0']}

Final result:

There are 1 misconfiguration parameters in the input: fs.s3a.connection.timeout
[Ciri] Reason for fs.s3a.connection.timeout: The value of fs.s3a.connection.timeout is not a valid integer.
